README.md

```diff
@@ -9,8 +9,8 @@ Terraform provider plugin for proxmox
 ## Go Install
 
 ```
-go install github.com/Telmate/terraform-provider-proxmox/cmd/terraform-provider-proxmox
-go install github.com/Telmate/terraform-provider-proxmox/cmd/terraform-provisioner-proxmox
+go install github.com/Telmate/terraform-provider-proxmox/tree/master/cmd/terraform-provider-proxmox
+go install github.com/Telmate/terraform-provider-proxmox/tree/master/cmd/terraform-provisioner-proxmox
```

Comment:
This isn't right. `go install` doesn't work this way and neither is `tree/master` in the path when a repo is cloned with `git`